infrastructure/cdn-in-a-box/readiness/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+start_time=$(date +%s)
+
+source to-access.sh
+
+set-dns.sh
+insert-self-into-dns.sh
```

Comment:
This is intended to be a short-lived container that other components don't make requests to, right? So why does it need to insert itself into the DNS records of the dns container?

Reply:
Monkey see monkey do. The component I was looking at was doing this with no specific reason to be in DNS. I don't think it really hurts anything to have all the components be found in DNS for consistency.

Reply:
I guess not.

What component was that?

Reply:
The TO API integration tests